src/checkboxes/checkboxes.test.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * @jest-environment ./lib/puppeteer/environment.js
+ */
+/* eslint-env jest */
+
+const cheerio = require('cheerio')
+
+const configPaths = require('../../config/paths.json')
+const PORT = configPaths.ports.test
+
+let browser
+let page
+let baseUrl = 'http://localhost:' + PORT
+
+const goToAndGetComponent = async (name, example) => {
+  const componentPath = `${baseUrl}/components/${name}/${example}/preview`
+  await page.goto(componentPath, { waitUntil: 'load' })
+  const html = await page.evaluate(() => document.body.innerHTML)
+  const $ = cheerio.load(html)
+  return $
+}
+
+const waitForHiddenSelector = async (selector) => {
+  return page.waitForSelector(selector, {
+    hidden: true,
+    timeout: 1000
+  })
+}
+
+const waitForVisibleSelector = async (selector) => {
+  return page.waitForSelector(selector, {
+    visible: true,
+    timeout: 1000
+  })
+}
+
+beforeEach(async () => {
+  browser = global.__BROWSER__
+  page = await browser.newPage()
+})
+
+afterEach(async () => {
+  await page.close()
+})
+
+describe('Checkboxes', () => {
+  describe('must be visible as static content if JavaScript is unavaliable or fails', () => {
```

Comment:
It'd be good to nail these describe and it blocks – it's unclear to me reading this _what we're trying to test_ and what is _context_.

For example 'with no aria attributes' and 'with visible content' as currently written could easily be interpreted as context ('when the thing does not have aria attributes', 'when the thing has visible content').

I'd suggest this would be better as something _like_:

```
Describe checkboxes with conditional reveals
  Describe when Javascript is unavailable or fails
    It does not have any ARIA attributes
    It makes the conditional content visible
```

The benefit of this structure is that the nesting still 'works', but each line also stands on its own – `describe` blocks read as context and `it` blocks read as assertions.

Reply:
Also, typo on unavaliable (appears a few times)

Reply:
yeah like I said above, I'm cleaning these up after the review with alex and jani